Tools/TestWebKitAPI/Tests/WebKitGtk/TestContextMenu.cpp

```diff
@@ -326,7 +327,7 @@ class ContextMenuDefaultTest: public ContextMenuTest {
     {
     }
 
-    bool contextMenu(WebKitContextMenu* contextMenu, GdkEvent* event, WebKitHitTestResult* hitTestResult)
+    bool contextMenu(WebKitContextMenu* contextMenu, GdkEvent*, WebKitHitTestResult* hitTestResult)
```

Comment:
Careful, this only works because we don't have bots running the API tests with the modern API. You have to do:

```
#if USE(GTK4)
bool contextMenu(WebKitContextMenu* contextMenu, WebKitHitTestResult* hitTestResult)
#else
bool contextMenu(WebKitContextMenu* contextMenu, GdkEvent*, WebKitHitTestResult* hitTestResult)
#endif
```

Reply:
Right!

Reply:
Actually, we have all tests in this file running only for gtk3. I think we need a new bug to enable these tests for gtk4.